Source/WebCore/editing/TextListParser.cpp

```diff
@@ -57,16 +57,19 @@ namespace WebCore {
 // MARK: Helpers
 
 template<typename Character>
-constexpr int consumeNumber(StringParsingBuffer<Character>& input)
+constexpr std::optional<int> consumeNumber(StringParsingBuffer<Character>& input)
 {
     // Parse the digits until there is no more input left or a non-ASCII digit character has been encountered.
-    Checked<int> value;
+    Checked<int, RecordOverflow> value;
     do {
         auto c = input.consume();
         int digitValue = c - '0';
         value = (value * 10) + digitValue;
     } while (!input.atEnd() && WTF::isASCIIDigit(*input));
 
+    if (value.hasOverflowed())
```

Comment:
does this need to be checked in the while loop condition instead? I forget what happens if you try to do arithmetic on an overflowed Checked value